ci/playbooks/kuttl/deploy-deps.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Run ci_framework bootstrap playbook
+  ansible.builtin.import_playbook: "../../../playbooks/01-bootstrap.yml"
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  name: install dev tools
+  tasks:
+    - name: Download install_yamls deps
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_download_tools'
+
+- name: Run ci_framework infra playbook
+  ansible.builtin.import_playbook: "../../../playbooks/02-infra.yml"
+
+- name: Build dataset hook
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Assert that operator_name is set
+      ansible.builtin.assert:
+        that:
+          - operator_name is defined
+
+    - name: Load parameters
+      ansible.builtin.include_vars:
+        dir: "{{ item }}"
+      loop:
+        - "{{ cifmw_basedir }}/artifacts/parameters"
+        - "/etc/ci/env"
+      loop_control:
+        label: "{{ item }}"
+
+    - name: Check we have some compute in inventory
+      ansible.builtin.set_fact:
+        has_compute: >-
+          {% set ns = namespace(found=false) -%}
+          {% for host in hostvars.keys() -%}
+          {%   if host is match('^compute.*') -%}
+          {%     set ns.found = true -%}
+          {%   endif -%}
+          {% endfor -%}
+          {{ ns.found }}
+
+    - name: Ensure that the isolated net was configured for crc
+      ansible.builtin.assert:
+        that:
+          - crc_ci_bootstrap_networks_out is defined
+          - "'crc' in crc_ci_bootstrap_networks_out"
+          - "'default' in crc_ci_bootstrap_networks_out['crc']"
+
+    - name: Ensure we have needed bits for compute when needed
+      when:
+        - has_compute | bool
+      ansible.builtin.assert:
+        that:
+          - "'compute' in crc_ci_bootstrap_networks_out"
```

Comment:
Why enforcing the instance name? I mean, you already used a different criteria to check if a compute is available, based on a regex. I'd vote to stick to a fixed name or pick the first one that is called `^compute.*`.

Reply:
same as above, removing this tasks as we won't have computes for kuttl